src/helpers/ens.ts

```diff
@@ -210,16 +260,25 @@ const getENSBaseRegistrarImplementationContract = (wallet?: Wallet) => {
   );
 };
 
+const getENSRegistryContract = () => {
+  return new Contract(
+    ensRegistryAddress,
+    ENSRegistryWithFallbackABI,
+    web3Provider
```

Comment:
just as an FYI (not needed to change right now before merge) - but we are trying to move away from directly using `web3Provider` and instead use `getProviderForNetwork` (also from `handlers/web3.ts`)

Reply:
cool i'm using it now 👌